pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/StudyRoleResponse.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsm.service.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class StudyRoleResponse {
+    private List<Role> roles;
+
+    public StudyRoleResponse(List<Role> roles) {
```

Comment:
We can use @AllArgsConstructor and get rid of this constructor . Initialize roles in the declaration